src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -1896,7 +1896,7 @@ public function testIfNotifierTransportsAreKnownByFrameworkExtension()
 
         $container = $this->createContainerFromFile('notifier');
 
-        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude('Mercure') as $bridgeDirectory) {
+        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude(['Mercure', 'FortysixElks']) as $bridgeDirectory) {
```

Comment:
why is this required?

Reply:
Transport is named `46elks` but classes are named `FortysixElks` (classnames cannot start with numbers).